src/components/Tool/MergePeople/MergePeople.test.tsx

```diff
@@ -138,10 +146,36 @@ describe('Tools - MergePeople', () => {
     userEvent.click(queryAllByTestId('rightButton')[0]);
     expect(await findByText('Use this one')).toBeInTheDocument();
     userEvent.click(queryAllByTestId('ignoreButton')[0]);
+    userEvent.click(queryAllByTestId('ignoreButton')[1]);
     expect(queryByText('Use this one')).not.toBeInTheDocument();
-    expect(
-      getByRole('button', { name: 'Confirm and Continue' }),
-    ).not.toBeDisabled();
+
+    userEvent.click(getByRole('button', { name: 'Confirm and Continue' }));
+    await waitFor(() =>
+      expect(mockEnqueue).toHaveBeenCalledWith('Updated 2 duplicate(s)', {
+        variant: 'success',
+      }),
+    );
+
+    const mergeCalls = mutationSpy.mock.calls
+      .map(([{ operation }]) => operation)
+      .filter(({ operationName }) => operationName === 'MergePeopleBulk');
+    expect(mergeCalls).toHaveLength(0);
+
+    const ignoreCalls = mutationSpy.mock.calls
+      .map(([{ operation }]) => operation)
+      .filter(({ operationName }) => operationName === 'UpdateDuplicate');
+    expect(ignoreCalls).toHaveLength(2);
+    expect(ignoreCalls[0].variables).toEqual({
+      input: {
+        attributes: {
+          ignore: true,
+        },
+        type: TypeEnum.Person,
+        id: '1',
+      },
+    });
```

Comment:
Make sure your testing clicking on mulitple contacts multiple times to ensure the correct contact Ids are set to the correct action.